crates/holochain/src/core/ribosome/host_fn/capability_info.rs

```diff
@@ -1,15 +1,14 @@
 use crate::core::ribosome::error::RibosomeResult;
 use crate::core::ribosome::CallContext;
 use crate::core::ribosome::RibosomeT;
-use holochain_types::prelude::*;
 use std::sync::Arc;
 
 /// return the access info used for this call
 /// also return who is originated the call (pubkey)
 pub fn capability_info(
     _ribosome: Arc<impl RibosomeT>,
     _call_context: Arc<CallContext>,
-    _input: CapabilityInfoInput,
-) -> RibosomeResult<CapabilityInfoOutput> {
+    _input: (),
```

Comment:
see above

Reply:
this will panic if anyone tries to use it, so a placeholder doesn't cause problems

```rust
pub fn capability_info(
    _ribosome: Arc<impl RibosomeT>,
    _call_context: Arc<CallContext>,
    _input: (),
) -> RibosomeResult<()> {
    unimplemented!();
}

```